### ISSUE TITLE
Add a more detailed documentation about Adding Error Messages

### ISSUE THREAD

**Issue by jellypuno** [zowe/vscode-extension-for-zowe#1224]
Relataed to #388

**Comment by JillieBeanSim** [comment]
@jellypuno @katelynienaber should this item be linked to the PR that will include the code and docs update? 
https://github.com/zowe/vscode-extension-for-zowe/pull/1433

**Comment by katelynienaber** [comment]
@JillieBeanSim Yes I think I will change the docs to agree with that PR, once the technical implementation is approved by everyone.

I will add the doc changes to that PR so I think this issue should be closed when that one is merged...

**Comment by JillieBeanSim** [comment]
Thanks @katelynienaber I think adding the docs/examples to the PR will help the reviewers test it
